src/elfpy/pricing_models.py

```diff
@@ -27,14 +27,14 @@ class PricingModel:
     # TODO: set up member object that owns attributes instead of so many individual instance attributes
     # pylint: disable=too-many-instance-attributes
 
-    def __init__(self, verbose=None):
+    def __init__(self, verbose=False):
```

Comment:
Please change this back to `None`. https://docs.python-guide.org/writing/gotchas/